components/canvas/Canvas.module.css

```
@@ -0,0 +1,13 @@
+.containermap {
+  background-image: url(/Albert.svg);
+  background-repeat: no-repeat;
+  align-content: center;
+}
+
+.show {
+  display: none;
+}
+
+.backgroundcanvas {
```

Use camelcase to make classes better readable
```suggestion
.backgroundCanvas {
```